Opserver.Core/Data/Dashboard/Providers/WmiDataProvider.Data.cs

```diff
@@ -136,6 +136,9 @@ public override Task<List<GraphPoint>> GetUtilizationAsync(Volume volume, DateTi
             return Task.FromResult(new List<GraphPoint>());
         }
 
+        public async Task<ServiceActionResult> UpdateServiceAsync(Node node, string serviceName, Data.Dashboard.NodeService.Action action) =>
+            await GetWmiNodeById(node.Id).UpdateServiceAsync(serviceName, action).ConfigureAwait(false);
```

Comment:
I'll fix after merge but tip: no need for `async`/`await` here - can just return the `Task` :)